packages/vitest/src/node/plugins/index.ts

```diff
@@ -47,6 +48,23 @@ export async function VitestPlugin(options: UserConfig = {}, ctx = new Vitest())
         // viteConfig.test is final now, merge it for real
         options = deepMerge(options, viteConfig.test as any || {})
         options.api = resolveApiConfig(options)
+
+        process.env.BASE_URL ??= viteConfig.base
+        process.env.MODE ??= viteConfig.mode
```

Comment:
I am not so sure about the `MODE` convention here, it feels too general to me. Do we have any references for other tools using it? or we better use `VITEST_MODE` or `VITE_MODE`

Reply:
This is what Vite uses on `import.meta.env`

Reply:
All these variables are default values on `import.meta.env` that always exist

Reply:
And we are allowing people to reassign them by making `import.meta.env` = `process.env`

we can assign them to `process.metaEnv` if you don't want to collaps with other tools, but I know this approach is wildly used now to support testing Vite in jest